en/developer-docs/docs/develop-components/develop-proxy/develop-an-api-proxy-from-an-open-api-specification.md

```diff
@@ -0,0 +1,128 @@
+# Develop an API Proxy from Open API Specification
+
+An API proxy acts as an intermediary between an existing API and Choreo, intercepting all requests made to the API. It also functions as a managed API, allowing you to apply essential API management features such as security policies and rate limiting.
+
+In this guide, you will:
+
+- Create an API proxy component to expose an existing API.
+- Deploy the API proxy.
+- Test the API proxy to verify its functionality.
+- Manage the API.
+- Consume the API.
+
+## Prerequisites
+
+- If you're signing in to the Choreo Console for the first time, create an organization:
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your preferred method.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the **Project Home** page of the default project created for you.
+
+## Step 1: Create an API proxy
+
+To create an API proxy, you can either upload an OpenAPI specification or provide an OpenAPI specification URL. In this guide, you will specify a URL to an OpenAPI definition of a sample API.
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
+2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
+3. Click the **API Proxy** card. This opens the **Create an API Proxy** pane.
+4. Select **Upload API Specification** option. 
+5. Click on **Try with sample URL** to go ahead with the sample Open API Specification.
+
+    !!! info
+        To streamline the process, if you already have a specification file saved on your device, simply upload it directly instead of relying on the sample.
+
+6. Specify the following values as Proxy Metadata:
+
+    !!! note
+        Values will auto populate according to the data in the uploaded/selected specification file. Following values are for the sample provided by choreo.
+
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Context**     | `api/v3`                                   |
```

Comment:
Shall we ask the user to change the context as the default one is a bit untidy? (it doesn't contain the API name and it has an additional version part as well)
I know its a little bit imperfect to ask for changing, but I feel its not good to move forward with the default value. suggestion: `swagger-petstore`
We might need to slightly alter the previous sentence as well, as we are changing the context - `Following values are for the sample provided by choreo.`

Reply:
https://github.com/wso2/docs-choreo-dev/pull/1788/files#r2006992742